x/reward/keeper/reward_distribution_test.go

```diff
@@ -321,11 +353,26 @@ func TestKeeper_DistributeRewards(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			var (
+				previousRewardPool     types.RewardPool
+				found                  bool
+				previousInitialCoins   sdk.Coins
+				previousRemainingCoins sdk.Coins
+			)
+
 			// set test reward pool if contains coins
-			if tt.rewardPool.Coins != nil {
+			if tt.rewardPool.RemainingCoins != nil {
 				k.SetRewardPool(ctx, tt.rewardPool)
-				err := bk.MintCoins(ctx, types.ModuleName, tt.rewardPool.Coins)
+				err := bk.MintCoins(ctx, types.ModuleName, tt.rewardPool.RemainingCoins)
 				require.NoError(t, err)
+
+				previousRewardPool, found = k.GetRewardPool(ctx, tt.args.launchID)
+				require.True(t, found)
+				previousInitialCoins = previousRewardPool.InitialCoins
+				previousRemainingCoins = previousRewardPool.RemainingCoins
```

Comment:
I think directly assigning the values from `tt.rewardPool` is sufficient since `SetRewardPool` is a method already tested